### ISSUE TITLE
bug:  buttons for modifying vlc playlist in vlc source plugin lac screen reader accessible labels 

### ISSUE THREAD

**Issue by marrie** [obsproject/obs-studio#8596]
### Operating System Info

Windows 11

### OBS Studio Version

29.0.2

### OBS Studio Log URL

https://obsproject.com/logs/5RoHGXRO8U0oU92T

### Expected Behavior

As a new beginner to this plugin, I should be able to, when tabbing with my screen reader on, be able to hear the plugin's button labels.

### Current Behavior

When tabbing around the vlc source plugin the buttons are not labeled correctly with my [screen reader](http://nvda-project.org). I hear “button enter

### Steps to Reproduce

1.  launch [NVDA](http://nvda-project.org). If not installed, download and install using the link given.
2.  launch OBS
3.  add the vlc source plugin to a scene.
4.  hit the applications key, arrow up to properties or configure it right away after adding.
5.  tab to the list of items. Notice it is empty.
6.  notice the button when you tab past that is called "button enter." this appears to be "add.
7.  next appears to be delete
8.  next appears to be edit.
9.  next appears to be move up.
10. next appears to be move down (I think I reversed those when asking for help in what these buttons were) These buttons are unlabeled, so as a new beginner I have to guess, a lot. This is counter intuitive when using the product.
